nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html

```diff
@@ -1 +1,90 @@
-More information about volpiano goes here
\ No newline at end of file
+<div class="container-fluid">
+    <div class="row">
+        <p>
+            Volpiano is a font that was created for the transcription of chant
+            melodies. It was developed Fabian Weber at the University of
+            Regensburg under the direction of David Hiley.
+        </p>
+        <p>
+            With Volpiano, alphanumeric characters and other symbols are
+            rendered as pitches and other musical elements on standard staffs.
```

Comment:
the pedant in me is impelled to assert that "staves" is the plural of "staff". But "staffs" is probably clearer than "staves" in this situation.